contracts/contracts/rem.swap/include/rem.swap/rem.swap.hpp

```diff
@@ -116,46 +112,42 @@ namespace eosio {
                         const block_timestamp &swap_timestamp, const signature &sign);
 
       /**
-       * Set swap fee action.
+       * Set token swap contract parameters action.
        *
-       * @details Change amount of the swap fee, action permitted only for producers.
+       * @details Change token swap contract parameters, action permitted only for producers.
        *
-       * @param quantity - the quantity of tokens to be deducted.
+       * @param in_swap_fee - fee for token swap in remchain,
+       * @param chain_id - the chain identifier of the remchain,
+       * @param eth_swap_contract_address - the address of the ethereum token swap contract,
+       * @param eth_return_chainid - the chain identifier of the ethereum network for out token swap.
        */
       [[eosio::action]]
-      void setswapfee(const int64_t &amount);
+      void setswapparam(const int64_t &in_swap_fee, const string &chain_id,
```

Comment:
make in_swap_fee separate for every chain_id

Reply:
@Art17, done.